frontend/components/bottom-buttons/js/bottom-buttons.component.ts

```diff
@@ -120,19 +124,34 @@ class BottomButtonsController implements ng.IController {
 		// Bind a click handler to exit focus mode
 		this.escapeFocusModeButton.addEventListener("click", this.focusMode.bind(this));
 
+		this.watchers();
+	}
+
+	public watchers() {
+		this.$scope.$watch(() => {
+			return this.ViewerService.getNavMode();
+		}, (newNav, oldNav) => {
+			if (newNav && newNav !== oldNav) {
+				if (newNav === Viewer.NAV_MODES.HELICOPTER) {
```

Comment:
Let's define the NAV_MODES as "Helicopter" and "Turntable" in Viewer.NAV_MODES

Then we can just do 
````js 
this.selectMode = newNav
````
And It will also auto fix the problem we have where Unity viewer is actually passing back "Helicopter" not "HELICOPTER"